### ISSUE TITLE
MuteSoundFile() doesn't stop currently playing sound instances

### ISSUE THREAD

**Issue by LenweSaralonde** [Stanzilla/WoWUIBugs#133]
`MuteSoundFile()` is a very convenient way to disable game sound that could be annoying for some players. However, it has no effect on instances of the sound that are currently playing:
```lua
/run PlaySoundFile(569593); MuteSoundFile(569593); -- Sound can still be heard
```
It would make sense that `MuteSoundFile()` also stops playing instances, especially for the looping ones.

**Comment by Meorawr** [comment]
I mean the solution here just seems to be to _not play the sound you're going to mute_. Realistically can you provide any actual use case where the muted file list won't be initialized on startup and, thus, wouldn't work as-is?

**Comment by LenweSaralonde** [comment]
`MuteSoundFile()` affects all game sounds, not only the ones played by the UI. Indeed, for UI sounds, the easiest trick is not to play them in the first place.

**Comment by Meorawr** [comment]
Yes I'm aware of what the function can be used for - I'm asking what the use case is for this feature.

Generally this function will be used with a preconfigured list of sound files to be muted and, as such, will be called ~around the time of `ADDON_LOADED` firing.

Does there exist a circumstance ingame at present where muting a sound file during that event fails to actually mute a file, or is this just intended to solve the situation of wanting to mute a file without a `/reload`?

**Comment by LenweSaralonde** [comment]
> this just intended to solve the situation of wanting to mute a file without a `/reload`

This. This is actually more an enhancement/feature request than a bug.
